File: java/sumofDigit.java
import java.util.Scanner;
class get_digit
{
	int n,c,sum=0;
	Scanner sc = new Scanner(System.in);
	void cal()
	{
		System.out.println("Enter the numbers: ");
		n=sc.nextInt();
		while(n>0)
		{
			c=n%10;
			sum=sum+c;
			n=n/10;
		}
		System.out.println("Sum of digits:"+sum);
	}
}

class sumofDigit
{
	public static void main(String args[])
	{
		get_digit obj = new get_digit();
		obj.cal();
	}
}

File: c language/sum of digit.c
#include<stdio.h>
main()
{
	int n,x,s=0;
	printf("\n Enter the value of N:");
	scanf("%d",&n);
	while(n>0)
	{
		x=n%10;
		s=s+x;
		n=n/10;
	}
	printf("\n sum is = %d",s);
}
